Vagrantfile

```diff
@@ -66,18 +69,23 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
 
   config.vm.network :private_network, ip: '192.168.50.50'
-  config.vm.synced_folder ".", "/vagrant", disabled: true
-  config.vm.synced_folder ".", "/vagrant-nfs", type: :nfs
+
+  if devconfig['use_nfs']
+      config.vm.synced_folder ".", "/vagrant", type: :nfs, nfs_udp: false
+  else
+      config.vm.synced_folder ".", "/vagrant"
+  end
+  
   config.bindfs.default_options = {
     force_user:   'ubuntu',
     force_group:  'ubuntu',
     perms:        'u=rwX:g=rD:o=rD'
   }
-  config.bindfs.bind_folder "/vagrant-nfs", "/home/ubuntu/intranet",
+  config.bindfs.bind_folder "/vagrant", "/home/ubuntu/intranet",
       force_user: 'ubuntu',
       force_group: 'ubuntu'
-  config.nfs.map_uid = Process.uid
-  config.nfs.map_gid = Process.gid
+  #config.nfs.map_uid = Process.uid
```

Comment:
Should this be dependent on the `use_nfs` flag instead of being commented out?

@naitian might know more about what is necessary for NFS to work correctly.